rules/tailwind.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  extends: [
+    '@edenspiekermann/stylelint-config/rules/tailwind',
+  ],
+  rules: {
+    // Disallow unknown at-rules.
+    // https://stylelint.io/user-guide/rules/at-rule-no-unknown
+    'at-rule-no-unknown': [null],
```

Comment:
To disable this rule we need to set...
```suggestion
    'at-rule-no-unknown': null,
```